app/views/manuals/_content_for_head_tag.html.erb

```diff
@@ -2,6 +2,7 @@
   <% if description.present? %>
     <meta name='description' content='<%= description %>' />
   <% end %>
+  <%= ab_test.requested_variant.analytics_meta_tag.html_safe if add_ab_meta_tag %>
```

Comment:
We discussed this in person, and we think we can't create a variant for whether a page is A/B tested or not, because it would conflict with the old/new variant.

I agree that the boolean parameter is ugly but it'll be removed at the end of the A/B test.